learntools/machine_learning/nbs/tut3.ipynb

```diff
@@ -0,0 +1,261 @@
+    "\n",
+    "We'll use the dot notation to select the column we want to predict, which is called the **prediction target**. By convention, the prediction target is called **y**. So the code we need to save the house prices in the Melbourne data is"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "_cell_guid": "e5a6c21c-cda7-4df8-a9e9-15a2ae865026",
+    "_uuid": "599dc0855ff966c6e8d84b5bd447b315bf85e339",
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "y = melbourne_data.Price"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "_cell_guid": "a52b0b8d-ea0a-435b-bde1-1472f63977ba",
+    "_uuid": "a0fac0889241e77f2e2deefbd63bc9f057d0c826"
+   },
+   "source": [
+    "# Choosing Predictors\n",
+    "Next we select the predictors. Sometimes, you will want to use all of the variables except the target. Other times you'll be better off with fewer variables. We'll build a simple model first and then you'll see how to iterate and compare different sets of predictors.\n",
+    "\n",
+    "We'll select multiple columns to use as predictors by providing a list of column names inside brackets. Remember, each item in that list should be a string (with quotes).\n",
+    "\n",
+    "Here is an example:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "_cell_guid": "b45e6964-5379-44b5-980c-11e556f34a4f",
+    "_uuid": "001553f51fbd9c115b7728e625fd6c2830852071",
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "melbourne_predictors = ['Rooms', 'Bathroom', 'Landsize', 'Lattitude', 'Longtitude']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "_cell_guid": "e30af6f8-45b8-4e33-9c3b-52a88266aee6",
+    "_uuid": "f9d9562f7d3a41042c3bb3beb64056993703735d"
+   },
+   "source": [
+    "By convention, this data is called **X**."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "_cell_guid": "f3ce77d5-06e2-4433-b392-16d54014d2c8",
+    "_uuid": "19b3d59d66644a58125cb8d41636a169d4c10af9",
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "X = melbourne_data[melbourne_predictors]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's quickly review the data we'll be using to predict house prices using the `describe` method and the `head` method, which shows the top few rows."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(X.describe())\n",
+    "print(X.head())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Visually checking your data with these commands is an important part of a data scientist's job.  You'll frequently find surprises in the dataset that deserve further inspection (e.g. to verify the data you have is correct)."
```

Comment:
I feel like this parenthetical belongs with the first sentence. (Or could just be cut)